systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -1022,7 +1026,11 @@ void testNotExistingCMSetsDefaultLogging() {
         KafkaResource.replaceKafkaResource(clusterName, kafka -> kafka.getSpec().getKafka().setLogging(
             new ExternalLoggingBuilder()
                 .withNewValueFrom()
-                    .withNewConfigMapKeyRef("log4j.properties", "not-existing-cm-name", false)
+                    .withConfigMapKeyRef(new ConfigMapKeySelectorBuilder()
+                            .withKey("log4j.properties")
+                            .withName("non-existing-cm-name")
```

Comment:
I am sorry, another typo
```suggestion
                            .withName("not-existing-cm-name")
```

Reply:
True